ibm/mas_devops/roles/cp4d_wds/tasks/embeddedwds.yml

```diff
@@ -0,0 +1,311 @@
+
```

Comment:
Does this support both cpd 3.5 and 4.0?

Reply:
Also, there are lots of linting failures that need to be fixed: https://github.com/ibm-mas/ansible-devops/runs/4657915346?check_suite_focus=true